ExtensionPack/README.md

```diff
@@ -0,0 +1,10 @@
+# C/C++ Extension Pack
+
+This Extension pack includes a set of popular extensions for C++ development in Visual Studio Code. It includes the following set of Visual Studio Code extensions:
+* [C/C++](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools) (Microsoft)
+* [C/C++](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools-themes) Themes (Microsoft)
+* [Cmake Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools) (Microsoft)
```

Comment:
CMake

Reply:
I think we can remove the second sentence and add a colon after the first.

"This extension pack includes a set of popular extensions for C++ development in Visual Studio Code:"